### DISCUSSION TITLE
Feature to show results only from specific provider?

### DISCUSSION THREAD

**Discussion by rwjack** [music-assistant/support#3119] [Feature requests and ideas]
I was looking for a specific mix on youtube (music), and when I made the search in MA, I also had a bunch of Spotify songs pollute my search results, when I specifically wanted to search through YT Music. Can we have advanced search options that allow hiding/showing results from specific providers?

**Comment by OzGav** [comment]
Duplicate https://github.com/orgs/music-assistant/discussions/2064
